files/en-us/web/api/fetch_api/using_fetch/index.html

```diff
@@ -35,9 +35,9 @@
   .then(data =&gt; console.log(data));
 </pre>
 
-<p>Here we are fetching a JSON file across the network and printing it to the console. The simplest use of <code>fetch()</code> takes one argument — the path to the resource you want to fetch — and returns a promise containing the response (a {{domxref("Response")}} object).</p>
+<p>Here we are fetching a JSON file across the network and printing it to the console. The simplest use of <code>fetch()</code> takes one argument — the path to the resource you want to fetch — and does not directly return the JSON response body but instead returns a promise that resolves with a {{domxref("Response")}} object.</p>
 
-<p>This is just an HTTP response, not the actual JSON. To extract the JSON body content from the response, we use the {{domxref("Response.json()")}} method.</p>
+<p>The {{domxref("Response")}} object, in turn, does not directly contain the actual JSON response body but is instead a representation of the entire HTTP response. So, to extract the JSON body content from the {{domxref("Response")}} object, we use the {{domxref("Response.json()", "json()")}} method, which resolves with a promise that’s the result of parsing the response body text as JSON.</p>
```

Comment:
I'm not feeling comfortable with promises, but this is my understanding. Am I wrong?

```suggestion
<p>The {{domxref("Response")}} object, in turn, does not directly contain the actual JSON response body but is instead a representation of the entire HTTP response. So, to extract the JSON body content from the {{domxref("Response")}} object, we use the {{domxref("Response.json()", "json()")}} method, which returns a second promise that resolves with the result of parsing the response body text as JSON.</p>
```

Reply:
> I'm not feeling comfortable with promises, but this is my understanding. Am I wrong?

You are not wrong — good catch, thanks